grades/models.py

```diff
@@ -1,12 +1,20 @@
 from django.db import models
 from django.conf import settings
-# Each user can add, modify or remove course from is list
-# Each course contains coures id(default by Django), name, credinitials and grade
+from django.core.validators import MaxValueValidator, MinValueValidator
+
 
 
 class Course (models.Model):
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE,
```

Comment:
I believe this table is the list of courses given, and the grades table combines the user and choose for a grade.
If so, this table probably shouldn't have a user relation

Reply:
+1